apps/web/lib/features/integrations/calendar/table-time-sheet.tsx

```
@@ -342,7 +343,10 @@ export function DataTableTimeSheet({ data }: { data?: GroupedTimesheet[] }) {
 														taskNumberClassName="text-sm"
 													/>
 												</div>
-												<span className="flex-1">{task.project && task.project.name}</span>
+												<div className="flex items-center gap-2 flex-1">
+													{task.project && <ProjectLogo imageUrl={task.project.imageUrl as string} />}
+													<span className="font-medium">{task.project && task.project.name}</span>
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)